recipes/mms-python-client/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "mms-python-client" %}
+{% set version = "3.4.2.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "6855d99218ed7e219e37d6cb6628443ac3abeb1921c16e0f778b1aee3e625290" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - urllib3 >=1.15
+    - six >=1.10
+    - certifi
+    - python-dateutil
+  run:
+    - python
+    - urllib3 >=1.15
+    - six >=1.10
+    - certifi
+    - python-dateutil
+
+test:
+  imports:
+    - mms_python_client
+
+about:
+  home: http://openmbee.org
+  license: Apache 2.0
+  license_file: LICENSE
+  summary: Model Management Service
+  description: 'Model Management System (MMS) is a version control system for structured data. It exposes model information through RESTful web services that can be used for CRUD operations, branching,
+    and tagging of the model repository.'
+  doc_url: ''
```

Comment:
Can you add URLs for `doc_url` and `dev_url`? If not, the entries should be removed entirely.

Reply:
Removed the entries entirely.